content/post/kubedb-v2023.08.18/index.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Announcing KubeDB v2023.08.18
+date: "2023-08-19"
+weight: 14
+authors:
+- Raihan Khan
+tags:
+- cloud-native
+- cruise-control
+- dashboard
+- database
+- day-2-operations
+- elasticsearch
+- kafka
+- kubedb
+- kubernetes
+- mariadb
+- mongodb
+- mysql
+- opensearch
+- percona
+- percona-xtradb
+- pgbouncer
+- postgresql
+- prometheus
+- proxysql
+- redis
+- security
+---
+
+We are pleased to announce the release of KubeDB v2023.08.18. This post lists all the major changes done in this release since the last release. The release includes -
```

Comment:
Use redirect link:
We are pleased to announce the release of [KubeDB v2023.08.18](https://kubedb.com/docs/v2023.08.18/setup/).